### ISSUE TITLE
Ansi component doesn't return correct classes

### ISSUE THREAD

**Issue by twavv** [nteract/archived-desktop-app#4816]
**Application or Package Used**
Package: `ansi-to-react`

**Describe the bug**
The current implementation doesn't differentiate between foreground and background when using the `useClasses` option.

**To Reproduce**
```jsx
// \u001b[40 is the ANSI code for black background
<Ansi>
{"\u001b[40mhello\u001b[0m"}
</Ansi>
```
results in
```html
<span class="ansi-black">hello</span>
```

**Expected behavior**
Generated classes should have `-fg`/`-bg` suffixes (this is what Anser does when generating html with the `use_classes` option, but because you're using the JSON functionality, it only returns the colors).
```html
<span class="ansi-black-bg">hello</span>
```

This arguably might be an issue with Anser itself.

**Comment by twavv** [comment]
Happy to make a PR to fix this if it should be fixed here instead of in Anser itself.

**Comment by captainsafia** [comment]
Hello @travigd! Thanks for opening this issue.

I think the fix might have to be made here. I'm suspecting that we will need to make a modification to the [createClass function](https://github.com/nteract/nteract/blob/master/packages/ansi-to-react/src/index.ts#L30) in the ansi-to-react package. I suspect that we will have to concatenate the class name and the key name into a single string.

**Comment by twavv** [comment]
> I think the fix might have to be made here. I'm suspecting that we will need to make a modification to the createClass function in the ansi-to-react package. I suspect that we will have to concatenate the class name and the key name into a single string.

Yeah, the code change shouldn't be too much. If it's okay, I'd also like to add classes that correspond to other decorations (e.g., `ansi-bold`, `ansi-underline`, etc.). If that sounds okay, I can make the PR. Currently, I'm running a patched version of the component in my own codebase so if I could contribute it back upstream and get rid of that, I'd be happy to contribute. :^)

**Comment by captainsafia** [comment]
@travigd That sounds sensible. A contribution is welcome here. Thanks for offering to contribute the improvement.

**Comment by CrystallineCat** [comment]
@travigd I fixed the wrong classes while I was touching the file to fix #4815. Your additional classes would still be very welcome. :smile_cat:
